users/models.py

```diff
@@ -0,0 +1,13 @@
+from xml.etree.ElementTree import ProcessingInstruction
+from django.db   import models
+
+from core.models import TimeStampModel
+
+class User(TimeStampModel): 
+    name         = models.CharField(max_length=50)
+    email        = models.CharField(max_length=200, unique=True)
+    kakao_id     = models.BigIntegerField()
+    point        = models.DecimalField(max_digits=20 ,decimal_places=2, default=100000000)
```

Comment:
1. = 기준으로 정렬 할 때는 가장 긴 단어를 기준으로 한 칸만 스페이스 해주세요.
2. 카카오로 로그인 할 때 못 받는 정보는 null=True나 default 값을 주셔야 합니다.
```suggestion
class User(TimeStampModel): 
    name     = models.CharField(max_length=50)
    email    = models.CharField(max_length=200, unique=True)
    kakao_id = models.BigIntegerField()
    point    = models.DecimalField(max_digits=20 ,decimal_places=2, default=100000000)
```

Reply:
카카오 공식문서 확인해보니까 이메일을 필수동의로 설정하려면 검수가 필요하다고 하네요! 이메일값 null=True로 바꾸겠습니다~